main.go

```diff
@@ -96,12 +98,12 @@ func proxyEap(rtrInt string, wanInt string, promiscuous bool) {
 	// Wait until both subroutines exit
 	fmt.Printf("proxy started. router: %s, wan: %s\n", rtrInt, wanInt)
 	quit := make(chan int)
-	go proxyPackets(rtrInt, rtrConn, wanInt, wanConn)
-	go proxyPackets(wanInt, wanConn, rtrInt, rtrConn)
+	go proxyPackets(rtrInt, rtrConn, wanInt, wanConn, ignoreLogoff)
+	go proxyPackets(wanInt, wanConn, rtrInt, rtrConn, false)
```

Comment:
I think this should be `ignoreLogoff`

Reply:
The reason I did `false` instead of `ignoreLogoff` is that I only ever see EAPOLLogOff from the router to the ONT, never the other way around. Figured I should always set this to false when going from ONT to router to avoid doing any extra work. I'm finding it a bit hard to locate documentation (at least that I understand) that verifies this but it's been working for me so far. Additionally, the python code that inspired you on this seems to only ignore from the router I think: https://github.com/jaysoffian/eap_proxy/blob/master/eap_proxy.py#L635